packages/google_sign_in/google_sign_in/ios/Tests/GoogleSignInPluginTest.m

```diff
@@ -153,4 +153,22 @@ - (void)testRequestScopesReturnsTrueIfGranted {
   XCTAssertTrue([result boolValue]);
 }
 
+- (void)testHostedDomainIfMissed {
+  FlutterMethodCall *methodCall =
+      [FlutterMethodCall methodCallWithMethodName:@"init"
+                                        arguments:@{
+                                          @"signInOption" : @"SignInOption.standard",
+                                          @"hostedDomain" : [NSNull null],
+                                        }];
+
+  XCTestExpectation *expectation =
+      [self expectationWithDescription:@"expect hostedDomain equals nil"];
+  [self.plugin handleMethodCall:methodCall
+                         result:^(id r) {
+                           [expectation fulfill];
+                         }];
+  [self waitForExpectations:@[ expectation ] timeout:5];
+  XCTAssertTrue([self.mockSharedInstance.hostedDomain == nil]);
```

Comment:
This file was missed during a change in how we run tests, so was not actually being compiled and run. In trying to fix it I found that:
a) it doesn't compile, because of this line, and
b) when that's fixed, other unit tests crash (presumably because calling `init` changes global state)

I'm going to have to remove this test to get the file running again; if you could submit a PR with a version of the test that compiles, and runs successfully in the context of the whole test suite, that would be great.

Reply:
The same test has been added here: https://github.com/flutter/plugins/blame/master/packages/google_sign_in/google_sign_in/example/ios/RunnerTests/GoogleSignInTests.m#L128.